client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,66 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
+  static targets = ['total', 'label'];
+  static values = {
+    containerBody: { type: String, default: 'body' },
+    min: { type: Number, default: 0 },
+    find: {
+      type: String,
+      default: '.error-message,.help-critical',
+    },
+    total: { type: Number, default: 0 },
+  };
+
+  declare containerValue: string;
+  declare minValue: number;
+  declare findValue: string;
+  declare totalValue: number;
+
+  declare readonly activeClass: string;
+  declare readonly hasActiveClass: string;
+  declare readonly hasLabelTarget: string;
+  declare readonly hasTotalTarget: string;
+  declare readonly labelTarget: HTMLElement;
+  declare readonly totalTarget: HTMLElement;
+
+  connect() {
+    // console.log("connected aman world");
+    setTimeout(() => {
+      console.log('connected CountController after load. ');
+      this.count();
+    }, 2000);
+  }
+
+  count() {
+    const containerElement = document.querySelector(
+      this.containerValue || 'body',
```

Comment:
This will not work, you have `containerValue`, but above you have set the `values` property to `containerBody`.

In `static values` above, you will need to update to the following; `    container: { type: String, default: 'body' },`.

If you have a value of `container`, this will be available with the getter `this.containerValue`, if you have a value of `robots`, it will be available with the getter `this.robotsValue`. These are 100% intertwined and the naming needs to be considered when using in your methods.

This is why your counts are counting all content in all tabs, because it's defaulting to `'body'`.

Have a re-read of https://stimulus.hotwired.dev/reference/values

Reply:
oops that was a typo, sorry.